### ISSUE TITLE
Connectivity to subscription table

### ISSUE THREAD

**Issue by hype08** [vercel/nextjs-subscription-payments#47]
# Connectivity to subscriptions table

Hi Thor! Thank you for the excellent tutorial!

I was able to successfully deploy and setup products/pricing through Stripe and Vercel.

When testing a subscription transaction just now in live mode, i was able to process a credit card.

Customers and Users table on supabase updates updates stripe customer id and table id, but nothing gets updated on the Subscription table. 

I'd like to have my app behave accordingly based on the subscription table and its columns, but missing this integration, so just curious if i'm missing an important detail somewhere.. or if it needs to be implemented.

Appreciate your help very much!

**Comment by hype08** [comment]
i figured out my own problem, just wanted to answer it here!

tldr in vercel, don't forget to promote your build to promotion for the live stripe webhook to kick in. d'oh!
